python-frontend/src/test/java/org/sonar/python/semantic/SymbolUtilsTest.java

```diff
@@ -279,17 +276,33 @@ public void path_of() throws IOException, URISyntaxException {
     assertThat(pathOf(pythonFile)).isNull();
   }
 
-  @Nullable
-  private static ClassDef findClassDefinition(Tree tree, String name) {
-    if (tree.is(Tree.Kind.CLASSDEF)) {
-      ClassDef classDef = (ClassDef) tree;
-      if (classDef.name().name().equals(name)) {
-        return classDef;
-      }
-    }
-    return tree.children().stream()
-      .map(child -> findClassDefinition(child, name))
-      .filter(Objects::nonNull)
-      .findFirst().orElse(null);
+  @Test
+  public void first_parameter_offset() {
```

Comment:
Wondered why snake_case, then saw that the entire file is in snake case... Ok, never mind.